app/views/hyrax/base/show.html.erb

```diff
@@ -33,7 +33,7 @@
 
     <div class="panel panel-default">
       <div class="panel-heading">
-        <h3 class="panel-title"><%= t('hyrax.base.show.relationships') %></h3>
```

Comment:
Are the Hyrax locale's not loading?  It would be good to leave this and items below as I18n.t calls.

Reply:
They are not, I agree it would be nice but they threw errors for me, this is something I will look into